AIPscan/Reporter/templates/reports.html

```diff
@@ -6,17 +6,33 @@
 
 {% if storage_services %}
 
-    <div style="padding-left: 1em;">
-        <strong>Storage service:</strong>
-        <select id="ss" method="GET" action="/" class="form-control" style="width:auto;">
-            {% for ss in storage_services %}
-              {% if ss.id == storage_service.id %}
-                <option value="{{ ss.id }}" SELECTED>{{ ss.name }}</option>
-              {% else %}
-                <option value="{{ ss.id }}">{{ ss.name }}</option>
-              {% endif %}
-            {% endfor %}
-        </select>
+    <div class="report-opts" style="">
+        <div>
+            <strong>Storage service:</strong>
+            <select id="ss" method="GET" action="/" class="form-control" style="width:auto;">
+                {% for ss in storage_services %}
+                  {% if ss.id == storage_service.id %}
+                    <option value="{{ ss.id }}" SELECTED>{{ ss.name }}</option>
+                  {% else %}
+                    <option value="{{ ss.id }}">{{ ss.name }}</option>
+                  {% endif %}
+                {% endfor %}
+            </select>
+        </div>
+        <div>
+            <span>
+                <img class="icon" src="{{ url_for('static', filename='assets/calendar-icon.png') }}" />
+                Start date
+            </span>
+            <input type="text" id="startdate" value="{{ start_date }}">
+            </div>
+        <div>
+            <span>
+                <img class="icon" src="{{ url_for('static', filename='assets/calendar-icon.png') }}" />
+                End date
+            </span>
+            <input type="text" id="enddate" value="{{ end_date }}">
+        </div>
```

Comment:
OK good! I should have added a screencap since I did want to get some buy-in here before doing anything else. I'll do that and then capture for larger consensus-building.

Reply:
Sounds great, thanks!